### DISCUSSION TITLE
TypeSupport File body is not properly generated

### DISCUSSION THREAD

**Discussion by edwin98thompson** [jmmorato/openddsharp#236] [Q&A]
I have a project which has 4 IDLs within the IDL folder, I have just added a 5th one. 
I have run, clean and build and the build completed successfully with no warnings, however the type support file for the new IDL has an empty body and I cant seem to figure out why. Any reason for this and is there a solution? many thanks.

![image](https://github.com/jmmorato/openddsharp/assets/68071932/e7e94510-0214-4b08-8169-e5f713bfe0e0)

**Comment by jmmorato** [comment]
Could you provide the files you used or at least the content of the fifth one?
